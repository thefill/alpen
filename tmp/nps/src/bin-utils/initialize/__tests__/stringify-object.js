import stringifyObject from './../stringify-object'

const objectToStringify = {
  foo: 'a',
  bar: {
    baz: 'b',
  },
}

test('stringify object correctly', () => {
  const stringObject = stringifyObject(objectToStringify, '    ')
  expect(stringObject).toMatchSnapshot()
})
