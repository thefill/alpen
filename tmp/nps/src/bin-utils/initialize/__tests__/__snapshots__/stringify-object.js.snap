// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stringify object correctly 1`] = `
"
    foo: 'a',
    bar: {
      baz: 'b'
    }"
`;
